### DISCUSSION TITLE
word wrap

### DISCUSSION THREAD

**Discussion by sandrey1** [pdfcpu/pdfcpu#996] [General]
In `pdfcpu stamp`, `pdfcpu create` text word wrap only if align justify?

**Comment by hhrutter** [comment]
I do not completely understand your question but I believe the answer is yes.

**Reply by sandrey1** [reply]
Based on textAndAlignment.json:
```
"text": [
	{
		"value": "A very long text in one line. But the words wrap to another line will only be if the alignment is justify :(",
		"width": 200,
		"align": "left"
	}
]
```

**Comment by hhrutter** [comment]
Yes..

..but just to be sure, please post a screenshot and mark what you don't think is correct.

Thank you for using pdfcpu 💚

**Comment by hhrutter** [comment]
Closed due to inactivity.
